custom_components/recycle_app/api.py

```
@@ -10,10 +10,16 @@
 
 
 class FostPlusApi:
+    """FostPlus API client."""
+
     __session: Session | None = None
     __endpoint: str
 
     def initialize(self) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)